### PR TITLE
Validator duties

### LINKED ISSUE #569: Validator duties
implement validator duties for specific validator
data should contain
- last 10 attestations
- last 10 block proposals 
- upcoming duties

styling example https://www.figma.com/file/dt1Z45BJkDGOg63IdNYwbR/ChainGuardian?node-id=0%3A1
image

### PR DESCRIPTION
**Pull request by BeroBurny**
**Short description of work done**
Implement validator duties screen

### PR Checklist

- [x] I have run type check locally
- [x] I have run linter locally
- [x] I have run unit and integration tests locally
- [x] Rebased to master branch / merged master

### Changes

- implement duties db
- implement fetching duties
- implement duties table

image

### Issues

Closes #569

### CODE REVIEW

File: src/renderer/constants/dutyStatus.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+    // dont change order of values
+    unknown,
+    scheduled,
+    missed,
+    orphaned,
+    proposed,
+    attested,
+}
+
+const text = ["Unknown", "Scheduled", "Missed", "Orphaned", "Proposed", "Attested"];
```
Comment (mpetrunic) [inline src/renderer/constants/dutyStatus.ts:11]: It's a lot safer to have map DutyStatus => text but that said, enum can also have those text values

File: src/renderer/constants/dutyStatus.ts [hunk 2]
```diff
@@ -0,0 +1,12 @@
+export enum DutyStatus {
+    // dont change order of values
```
[the remark this answers is not in the document]
Comment (BeroBurny) [reply src/renderer/constants/dutyStatus.ts:2]: intentionally did that way, without values enum is numerated like an array (but want to keep names for easier readability) and for storing in DB it is more efficient of storing strings and have some logic based on numbers inside `put` logic

File: src/renderer/constants/dutyStatus.ts (hunk 1 above)
Comment (BeroBurny) [reply src/renderer/constants/dutyStatus.ts:11]: then I will use map

File: src/renderer/constants/dutyStatus.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (mpetrunic) [reply src/renderer/constants/dutyStatus.ts:2]: Why not have

enum DutyStatus {
   uknown: "Unknown"
}

?

[the remark this answers is not in the document]
Comment (BeroBurny) [reply src/renderer/constants/dutyStatus.ts:2]: https://github.com/NodeFactoryIo/ChainGuardian/blob/607e062cfc3c1253b425db35b5e221025e71584d/src/renderer/models/propositionDuties.ts#L25

https://github.com/NodeFactoryIo/ChainGuardian/blob/607e062cfc3c1253b425db35b5e221025e71584d/src/renderer/models/attestationDuties.ts#L25

**Review by mpetrunic — APPROVED** [review]

**Pull request merged**